core/meta/inc/TGlobal.h

```diff
@@ -48,36 +49,53 @@ class TGlobal : public TDictionary {
    ClassDef(TGlobal,2)  //Global variable class
 };
 
-   //Class to map the "funcky" globals and be able to add them to the list of globals.
-   class TGlobalMappedFunction: public TGlobal {
-   public:
-      typedef void* (*GlobalFunc_t)();
-      TGlobalMappedFunction(const char* name, const char* type,
-                            GlobalFunc_t funcPtr):fFuncPtr(funcPtr)
-      {
-         SetNameTitle(name, type);
-      }
-      virtual       ~TGlobalMappedFunction() {}
-      Int_t          GetArrayDim() const { return 0;}
-      DeclId_t       GetDeclId() const { return (DeclId_t)(fFuncPtr); } // Used as DeclId because of uniqueness
-      Int_t          GetMaxIndex(Int_t /*dim*/) const { return -1; }
-      void          *GetAddress() const { return (*fFuncPtr)(); }
-      const char    *GetTypeName() const { return fTitle; }
-      const char    *GetFullTypeName() const { return fTitle; }
-      Long_t         Property() const { return 0; }
-      virtual bool   Update(DataMemberInfo_t * /*info*/) { return false; }
-      static  void   Add(TGlobalMappedFunction* gmf);
-
-   private:
-
-      GlobalFunc_t fFuncPtr; // Function to call to get the address
-
-      TGlobalMappedFunction &operator=(const TGlobal &); // not implemented.
-      // Some of the special ones are created before the list is create e.g gFile
-      // We need to buffer them.
-      static TList&  GetEarlyRegisteredGlobals();
-
-      friend class TROOT;
-   };
+// Class to map the "funcky" globals and be able to add them to the list of globals.
+class TGlobalMappedFunction : public TGlobal {
+public:
+   typedef void *(*GlobalFunc_t)();
+   typedef std::function<void *()> GlobalFunctor_t;
+   TGlobalMappedFunction(const char *name, const char *type, GlobalFunc_t funcPtr) : fFuncPtr(funcPtr)
+   {
+      SetNameTitle(name, type);
+   }
+   virtual ~TGlobalMappedFunction() = default;
```

Comment:
Please keep the original alignment of function name :)

Reply:
I use here clang-format. Should I apply hand-made formatting?

Reply:
yes for that part. clang-format does not support this kind of alignment.